packages/ui/src/components/ButtonIcon/styles.css.ts

```diff
@@ -16,6 +16,7 @@ export const variants = recipe({
 			cursor: 'pointer',
 			padding: 0,
 			outline: 'none',
+			flexShrink: 0,
```

Comment:
Not directly related to this change, but prevents the square buttons from being squished and changing shape.